les/serverpool.go

```diff
@@ -96,6 +96,7 @@ var (
 	sfQueried          = serverPoolSetup.NewFlag("queried")
 	sfCanDial          = serverPoolSetup.NewFlag("canDial")
 	sfDialed           = serverPoolSetup.NewFlag("dialed")
+	sfDialTimeout      = serverPoolSetup.NewFlag("dialTimeout")
```

Comment:
Ah, `sfDialTimeout` seems not a good name. Let's rename it to `sfWaitDialTimeout`

Reply:
done